src/RoleObjects/AccessibilityObject.js

```diff
@@ -177,6 +177,117 @@ export default class AccessibilityObject {
     return this._reactProps.accessKey;
   }
 
+  /**
+   * The translate attribute specifies whether the content of an element should be translated or not.
+   * @access public
+   * @param {String} str - "yes" or "no" Specifies that the content of the element should be translated or not
+   */
+  set translate(str) {
```

Comment:
Since this attribute isn't supported in any major browser according to https://www.w3schools.com/tags/att_global_translate.asp, I think it's better to not provide a getter or setter for this so that we don't create the impression that CAM has found a way to support this when browsers don't.

Reply:
updated